tests/status.spec.coffee

```diff
@@ -90,7 +103,7 @@ describe 'Status', ->
 				key: 'offline'
 				name: 'Offline'
 
-		it 'should return UPDATING if there is a download_progress and status is Downloading and is_online is true', ->
+		it 'should return UPDATING if there is a download_progress and status is Downloadinga nd is_online is true', ->
```

Comment:
Typo in here

Reply:
Oh, how did that `a` migrate to the previous word? :alien: :satellite: